experimental/devices/pca9685/servo.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package pca9685
+
+// ServoGroup a group of servos connected to a pca9685 module
+type ServoGroup struct {
+	*Dev
+	minPwm   int
+	maxPwm   int
+	minAngle int
+	maxAngle int
+}
+
+// Servo individual servo from a group of servos connected to a pca9685 module
+type Servo struct {
+	group    *ServoGroup
+	channel  int
+	minAngle int
+	maxAngle int
+}
+
+// NewServoGroup returns a servo group connected throught the pca9685 module
```

Comment:
Please use https://periph.io/x/periph/conn/physic#Angle

That said, the angle-to-period logic is fairly device agnostic so the mapValue() code should probably live elsewhere. I started package https://periph.io/x/periph/experimental/devices/gpioutil for this kind of things. I plan to move it under conn/gpio/ at the end.

Reply:
The problem is: how to implement a generic `mapValue` function? Because was initially implemented for `int` values and now is a mix of ints and `physic.Angle`, so I don't know how can we make a more generic function that can be used in other places.